### DISCUSSION TITLE
Setting Element's Z-Axis Property Manually

### DISCUSSION THREAD

**Discussion by Fractal393** [clientIO/joint#2537] [Q&A]
### Introduction

I have two types of elements in my stencil, 1. Container, 2. Object.
What I want is for the object element to have the Z-axis property always higher than the Container element irrespective of when it is dropped. I have some ways to solve this but most of them don't seem efficient.

Is there a way to do this using pre defined methods?

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment]
The stencil is currently unsetting the `z` index from clones before dropping. You can set a new `z` after dropping an element.

```js
stencil.on('element:drop', (elementView) => {
    const element = elementView.model;
    element.set('z', isContainer(element) ? 1 : 2);    
});
```
